ovirtlago/cmd.py

```diff
@@ -174,13 +179,20 @@ def do_ovirt_reposetup(
         reposync_yum_config=reposync_config,
         skip_sync=skip_sync,
         custom_sources=custom_sources,
+        internal_repo_path=internal_repo_path,
     )
 
 
 @cli_plugin(help='Run scripts that install necessary RPMs and configuration')
+@cli_plugin_add_argument(
+    '--internal-repo',
+    help='Path to local rpm repository',
+    type=os.path.abspath,
+)
 @in_ovirt_prefix
 @with_logging
 def do_deploy(prefix, **kwargs):
+    prefix._set_internal_repo(kwargs['internal_repo'])
```

Comment:
Hmmm... I think that this might better be done in the constructor of the prefix instead of here (check the in_prefix function)